superset/datasets/commands/duplicate.py

```diff
@@ -44,60 +43,51 @@
 
 class DuplicateDatasetCommand(CreateMixin, BaseCommand):
     def __init__(self, data: Dict[str, Any]) -> None:
-        self._base_model: SqlaTable = SqlaTable()
+        self._base_model: Optional[SqlaTable] = None
```

Comment:
Changed to `Optional` since `find_by_id` could return `None`, and also it's more consistent how we handle such cases.